frontend/src/util/validators.js

```diff
@@ -194,6 +194,13 @@ export function validateMailingAddress() {
 export function validateMailingAddressForMSPContracts() {
   return this.mailingAddress.addressLine2 !== '' || this.mailingAddress.city !== '' || this.mailingAddress.province !== '' || this.mailingAddress.postalCode !== ''
 }
+
+/**
+ * Used to validate that Mailing Address line 1 must be completed if any other Mailing Address(Line 2, city, province, PostalCode) is complete
+ */
+export function validateMailingAddressForGroupMember() {
```

Comment:
What about postal code? Shouldn't be mandatory if city and province are there. Also, apply similar validation for addressline1